src/usermod.c

```diff
@@ -308,13 +308,13 @@ static struct ulong_range getulong_range(const char *str)
 	errno = 0;
 	first = strtoll(str, &pos, 10);
```

Comment:
Why not use `strtoul` or `strtoull` instead? This whole function looks bogus to the point of maliciousness.

Reply:
Indeed.

Reply:
`strtoul` or `strtoull` would silently accept negative input.

Reply:
Ahh, that seems true.